core-bundle/src/Image/PictureFactory.php

```diff
@@ -210,10 +216,14 @@ static function ($formatsString) {
                 $config->setFormats($imageSizes['formats'] ?? []);
                 $options->setSkipIfDimensionsMatch($imageSizes['skipIfDimensionsMatch'] ?? false);
 
-                if ($imageSizes && isset($imageSizes['cssClass']) && $imageSizes['cssClass']) {
```

Comment:
Did you intentionally remove the `$imageSizes` check here?

Reply:
Everything I do is intentionally 😂